projects/media-viewer/src/lib/annotations/comment-set/comment/comment.component.ts

```diff
@@ -1,32 +1,33 @@
-import { TagItemModel } from '../../models/tag-item.model';
-import { TagsServices } from '../../services/tags/tags.services';
 import {
+  AfterViewChecked,
   Component,
   ElementRef,
   EventEmitter,
   Input,
-  OnChanges,
   OnDestroy,
+  OnInit,
   Output,
   ViewChild
 } from '@angular/core';
-import { User } from '../../models/user.model';
-import { AnnotationEventService, SelectionAnnotation } from '../../annotation-event.service';
+import {Comment} from './comment.model';
+import {User} from '../../models/user.model';
+import {Rectangle} from '../../annotation-set/annotation-view/rectangle/rectangle.model';
+import {SelectionAnnotation} from '../../models/event-select.model';
+import {CommentService} from './comment.service';
+import {TagItemModel} from '../../models/tag-item.model';
+import {TagsServices} from '../../services/tags/tags.services';
 import { Subscription } from 'rxjs';
-import { CommentService } from './comment.service';
-import { debounceTime, distinctUntilChanged } from 'rxjs/operators';
-import { Rectangle } from '../../annotation-set/annotation-view/rectangle/rectangle.model';
-import { Comment } from './comment.model';
+import { distinctUntilChanged } from 'rxjs/operators';
+import {Store} from '@ngrx/store';
+import * as fromStore from '../../../store';
 
 @Component({
   selector: 'mv-anno-comment',
   templateUrl: './comment.component.html'
 })
-export class CommentComponent implements OnChanges, OnDestroy {
-
-  readonly MAX_COMMENT_LENGTH;
```

Comment:
is this no longer needed, or is it managed elsewhere? it seems like this is a good place to manage it, catch the error closest to where it happens

Reply:
it wasn't used so i just removed it